File: sdkwork-rtc-sdk-java/providers/rtc-sdk-provider-janus/src/main/java/com/sdkwork/rtc/provider/janus/RtcProviderJanusPackageContract.java
package com.sdkwork.rtc.provider.janus;

/**
 * Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
 */
public final class RtcProviderJanusPackageContract {
  public static final String PROVIDER_KEY = "janus";
  public static final String PLUGIN_ID = "rtc-janus";
  public static final String DRIVER_ID = "sdkwork-rtc-driver-janus";
  public static final String PACKAGE_IDENTITY = "com.sdkwork:rtc-sdk-provider-janus";
  public static final String STATUS = "future-runtime-bridge-only";
  public static final String RUNTIME_BRIDGE_STATUS = "reserved";
  public static final boolean ROOT_PUBLIC = false;

  private RtcProviderJanusPackageContract() {
  }
}

File: sdkwork-rtc-sdk-csharp/providers/Sdkwork.Rtc.Sdk.Provider.Janus/src/RtcProviderJanusPackageContract.cs
namespace Sdkwork.Rtc.Sdk.Provider.Janus;

/// <summary>
/// Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
/// </summary>
public static class RtcProviderJanusPackageContract
{
    public const string ProviderKey = "janus";
    public const string PluginId = "rtc-janus";
    public const string DriverId = "sdkwork-rtc-driver-janus";
    public const string PackageIdentity = "Sdkwork.Rtc.Sdk.Provider.Janus";
    public const string Status = "future-runtime-bridge-only";
    public const string RuntimeBridgeStatus = "reserved";
    public const bool RootPublic = false;
}
